### ISSUE TITLE
Hyperparameter Tuning of CNN

### ISSUE THREAD

**Issue by PXDHU** [Oxlac/Scribblr#2]
Tune the following hyperparameters for Convolutional Neural Network to achieve more accuracy:

- Learning rate
- Number of epochs
- Batch size
- Activation function
- Number of hidden layers and units

Visualization

- plot accuracy and loss graph
